f5-sdk-dist/scripts/README.rst

```diff
@@ -0,0 +1,77 @@
+f5 Package Creation and Automation
+==================================
+
+Introduction
+------------
+You've gon the wrong way!  This directory and lower directories are not
```

Comment:
The "You've gon the wrong way!" warning is cute, but maybe: "Have you gone the wrong way?!"